server/global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
There are several auto-gened wrappers. I believe that they allow addition of .xproj projects (tests in this case) to be linked with normal .csproj